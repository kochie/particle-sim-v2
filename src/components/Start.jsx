import React from 'react';

export default class Start extends React.component {
	render() {
		return (<div></div>)
	}
}