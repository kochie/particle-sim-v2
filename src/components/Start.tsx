import * as React from 'react';

export default class Start extends React.Component {
	public render(): React.ReactElement {
		return (<div></div>)
	}
}