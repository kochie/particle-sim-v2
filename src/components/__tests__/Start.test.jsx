import renderer from 'react-test-renderer';
import React from 'react';

import Start from '../Start';

function init(props) {
  const instance = renderer.create(<Start {...props} />);
  return { instance };
}

describe('start react component', () => {
  it('should render and match snapshot', () => {
    const { instance } = init();
    expect(instance).toMatchSnapshot();
  });
});
