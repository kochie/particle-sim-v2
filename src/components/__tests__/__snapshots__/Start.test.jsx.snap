// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`start react component should render and match snapshot 1`] = `<div />`;
